include/exadg/solvers_and_preconditioners/solvers/wrapper_elementwise_solvers.h

```diff
@@ -90,8 +90,6 @@ class IterativeSolver
   unsigned int
   solve(VectorType & dst, VectorType const & src) const override
   {
-    dst = 0;
-
     op.get_matrix_free().cell_loop(&THIS::solve_elementwise, this, dst, src);
```

Comment:
`dst` is only used in line 148 in the new code; the `dst=0` did not even have an effect I think.